support/Environments/setup/frontera.sh

```diff
@@ -109,7 +121,8 @@ else
     mkdir build
     cd build
     cmake -D CMAKE_BUILD_TYPE=Release -D YAML_CPP_BUILD_TESTS=OFF \
-          -D CMAKE_C_COMPILER=gcc -D CMAKE_CXX_COMPILER=g++ \
+          -D CMAKE_C_COMPILER="$c_compiler" \
+          -D CMAKE_CXX_COMPILER="$cxx_compiler" \
```

Comment:
Do you want to specify the compilers for charm and libsharp as well?

Reply:
No I'll leave them as is. For charm, unless there is a reason to use a specific compiler, I'd rather let charm decide. For libsharp, the [docs](https://github.com/Libsharp/libsharp#compilation) say it's only supported to build with GNU and Intel compilers, so it should automatically grab gcc as it's loaded for both.

Reply:
Can we just build everything with gcc in both cases then?  It would simplify managing modules if gcc and clang setups used the same directory.

Reply:
Let me give it a try and see if it works. If it does then you're right that would be easier.